commands/TransferIndexConfig.js

```diff
@@ -59,6 +59,7 @@ class TransferIndexConfigScript extends Base {
   async transferIndexConfig(indices, config) {
     // Transfer settings, synonyms, and query rules
     const settings = await indices.sourceIndex.getSettings();
+    delete settings['replicas'];
```

Comment:
```suggestion
    delete settings.replicas;
```

(suggestion from eslint)

Reply:
@Haroenv Thank you!